PowerPointLabs/PowerPointLabs/DefaultMotionAnimation.cs

```diff
@@ -9,12 +9,15 @@ namespace PowerPointLabs
     class DefaultMotionAnimation
     {
 #pragma warning disable 0618
+
+        private const float ArrowheadLength = 6.0f;

```

Comment:
Oh thanks for pointing out the bad practice. I realized that arrowhead can have different sizes, which causes the rotation pivot to be off from the center. The new commit fixes the pivot issue by grouping a transparent line in the opposite direction.